releasenotes/notes/bifrost-fix-ipa-auth-9d0e0c3b948b5850.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixes an issue where the IPA images in Release Train could not be
```

nit: I'd probably call it Ark here but that's personal preference so I'm happy to approve anyway
```suggestion
    Fixes an issue where the IPA images in Ark could not be
```

---

I'll take your opinions, thanks.